extensions/amp-story/1.0/page-advancement.js

```diff
@@ -708,7 +708,8 @@ export class ManualAdvancement extends AdvancementConfig {
     if (
       !this.isRunning() ||
       !this.isNavigationalClick_(event) ||
-      this.isProtectedTarget_(event) ||
+      (this.isProtectedTarget_(event) &&
```

Comment:
optional nit: from a readability perspective, it can be good to avoid mixing `&&`'s and `||`'s, since it makes the order of operations harder to parse